src/ContentRepository/Fields/ReferenceFieldSetting.cs

```diff
@@ -18,15 +18,14 @@ public class ReferenceFieldSetting : FieldSetting
         public const string AllowMultipleName = "AllowMultiple";
         public const string AllowedTypesName = "AllowedTypes";
         public const string SelectionRootName = "SelectionRoot";
-        public const string QueryName = "Query";
+        public const string QueryName = "Query"; //UNDONE:!!!! NODEQUERY

```

Comment:
Previously Query could be a NodeQuery expression. Maybe we could convert it to (or expect/allow) a ContentQuery text.